model_cards/jimregan/BERTreach/README.md

```diff
@@ -0,0 +1,21 @@
+---
+language: ga
+tags:
+- irish
+---
+
+## BERTreach
+
+([beirtreach](https://www.teanglann.ie/en/fgb/beirtreach) means 'oyster bed')
+
+**Model size:** 84M
+
+**Training data:** 
+* [PARSEME 1.2](https://gitlab.com/parseme/parseme_corpus_ga/-/blob/master/README.md) 
+* Newscrawl 300k portion of the [Leipzig Corpora](https://wortschatz.uni-leipzig.de/en/download/irish)
+* Private news corpus crawled with [Corpus Crawler](https://github.com/google/corpuscrawler)
```

Comment:
How large was the total dataset?

Reply:
2125804  sentences, 47419062 (wc) tokens. Pretty small, but a lot bigger than the one from Turku.

Reply:
Feel free to add to the model card! (we'll make it way easier to update in the next few weeks)